modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-classic-aws-rhcos-ami_{context}"]
+= ROSA AMIs for {product-title}
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {product-title} nodes.
+
+.{product-title} AMIs
+
+[cols="2a,2a,2a",options="header"]
+|===
+
+|AWS zone
+|AWS AMI
+|Description
+
+.19+|*us-east-1*
+|`ami-00bd5e71974ac1199`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-083b6a0085bfb83d7`
+|`rhcos-46.82.202008071940-0`
+|`ami-02325594e61f75251`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0a7427bf6706eed45`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?

Reply:
@iamkirkbater generated these for me, so I wasn't sure if that was a critical piece to include or not. I can remove those references if it's clear on its own enough.

Reply:
I think we can go ahead and remove that entire column if you restructure to include versions. The real thing that a user needs is the AMI ID.